addons/website_event/views/snippets/snippets.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+
+<template id="remove_external_snippets" inherit_id="website.external_snippets">
+    <xpath expr="//t[@t-install='website_event']" position="replace"/>
+</template>
+
+<template id="snippets" inherit_id="website.snippets">
+    <xpath expr="//t[@id='event_upcoming_snippet_hook']" position="replace">
+        <t t-snippet="website_event.s_events" t-thumbnail="/website/static/src/img/snippets_thumbs/s_event_upcoming_snippet.svg"/>
```

Comment:
@pko-odoo Problem introduced by this PR: a template with the ID "snippets" (website_event.snippets) already existed in this app... so this new one overrides the other. Result: the s_speaker_bio snippet is still defined in the app but is dead code. Probably the fix is just about moving the t-snippet="website_event.s_speaker_bio" here, probably stable ok/enough. Would you have a look? Thanks :+1: